Bottom: 88895270858f2dd2352540922445fa06acef1927
Top:    b07ba922d91a7054739a0bd458ae9b044cc76652
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 15:15:19 +0100

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/borast/README b/src/borast/README
new file mode 100644
index 0000000..b187cf2
--- /dev/null
+++ b/src/borast/README
@@ -0,0 +1,13 @@
+This directory contains Bentlely-Ottman routines and other support code copied
+(and modified) from the cairo graphics library (http://www.cairographics.org/)
+
+In order to keep the namespace clean, the file and function prefix cairo_ has been
+replaced with borast_. CAIRO_ constants have been replaced with BORAST_
+
+Cairo is free software and is available to be redistributed and/or
+modified under the terms of either the GNU Lesser General Public
+License (LGPL) version 2.1 or the Mozilla Public License (MPL) version
+1.1.
+
+The code in this directory is distributed under the Lesser General Public
+License (LGPL) version 2.1.
